src/Stryker.Core/Stryker.Core/StrykerRunner.cs

```diff
@@ -40,6 +44,15 @@ public StrykerRunResult RunMutationTest(StrykerOptions options)
             var stopwatch = new Stopwatch();
             stopwatch.Start();
 
+            // Create output dir with gitignore
+            string strykerOutputDirectory = Path.Combine(options.BasePath, "StrykerOutput");
+            _fileSystem.Directory.CreateDirectory(strykerOutputDirectory);
+            _fileSystem.File.Create(Path.Combine(strykerOutputDirectory, ".gitignore")).Close();
```

Comment:
Using instead of `.Close()`.
Or is this not needed? Doesn't CreateText create or update a file? So now it gets created twice.